### ISSUE TITLE
sqlite filer backend breaks on delete

### ISSUE THREAD

**Issue by tv42** [chrislusf/seaweedfs#2215]
```
$ weed version
version 30GB 2.59  linux **amd64
$ cat filer.toml
[sqlite]
enabled = true
dbFile = "./data-filer/filer.sqlite"
```

Copy some data under `/foo`. Mount the FUSE filesystem. Run `rm -rf mountpoint/foo/bar/quux`. Observe this error:

```
I0723 14:10:18 84299 dir.go:462] remove /foo/bar/quux: delete directory /foo/bar/quux: filer store delete: deleteFolderChildren /foo/bar/quux: SQL logic error: near "LIMIT": syntax error (1)
```

**Comment by chrislusf** [comment]
Thanks for reporting! This should have already been fixed in https://github.com/chrislusf/seaweedfs/commit/182288f8606bde8bfcb31509335a4c33802db254

You can use the dev version https://github.com/chrislusf/seaweedfs/releases/dev to confirm.

**Comment by kmlebedev** [comment]
> Thanks for reporting! This should have already been fixed in [182288f](https://github.com/chrislusf/seaweedfs/commit/182288f8606bde8bfcb31509335a4c33802db254)
> 
> You can use the dev version https://github.com/chrislusf/seaweedfs/releases/dev to confirm.
@chrislusf 
This is a fix 
```
tar xzf sqlite-3.6.20.tar.gz
cd sqlite-3.6.20
export CFLAGS='-DSQLITE_ENABLE_UPDATE_DELETE_LIMIT=1'
./configure
make
```

Or 
```
DELETE FROM Foo
WHERE someColumn in
(
  SELECT someColumn FROM FOO WHERE SomeCondition LIMIT 200
)
```

Please re-open

**Comment by chrislusf** [comment]
How about adding this option for mysql and postgres?
```
deletion_chunk_size=0
```
Default to zero to avoid impacting sqlite adapter, which reuses mysql adapter code.
